### DISCUSSION TITLE
Use `aws` instead of `awscc`

### DISCUSSION THREAD

**Discussion by jolo-dev** [stackql/stackql.io#119] [Ideas]
I think it is unclear for beginners to distinguish which services are available in `aws` and which are in `awscc`. I propose to consolidate everything into `aws`. What do you think?

**Comment by jeffreyaven** [comment]
Hi, mutable versions of core services (`ec2`, `s3`, `iam`) are in the `aws`provider, meaning they support creation, deletion, updating of resources as well as lifecycle methods like `aws.ec2.instances.start` and `stop`, `awscc` is built on the AWS Cloud Control API, at this stage it supports `SELECT| for all resources, but will ultimately support all SQL methods.  


To see services supposed on a provider you can use [__`SHOW SERVICES`__](https://stackql.io/docs/language-spec/show#list-all-services-in-a-cloud-provider).  


You can also see the provider docs [aws](https://registry.stackql.io/providers/aws/) or [awscc](https://registry.stackql.io/providers/awscc/). 


Let me know if this helps

**Comment by jolo-dev** [comment]
I understand. Then I would swap the `awscc` and `aws`. It's then like AWS CLI `s3` command where `s3api` offers simplified API with S3.

**Reply by jeffreyaven** [reply]
Sort of, at some stage we may merge both into one, the `awscc` is mainly views using the `cloud_control` service, what's your use case out of interest?

**Reply by jolo-dev** [reply]
I am trying out StackQL to create a dynamic AWS inventory, and I was a bit confused when I used it to get the Lambdas.

**Comment by jeffreyaven** [comment]
Try the cookbook, let me know if you have any problems, it can be adapted
to get more information about the function if you need this

**Reply by jolo-dev** [reply]
Thank you @jeffreyaven 
Yes, I used the cookbook. I also fully understand the code. Wanted to know about the decision why it is called `awscc`.

**Comment by jeffreyaven** [comment]
This is short for [AWS Cloud Control](
https://docs.aws.amazon.com/cloudcontrolapi/)

**Reply by jolo-dev** [reply]
I see. Make sense now :D
